package.json

```diff
@@ -42,14 +45,27 @@
     }
   },
   "dependencies": {
+    "@blueprintjs/core": "^4.6.0",
```

Comment:
Many of these new dependencies are already part of canon-core, is there a reason you needed to install them? They will add to the over all package size and make the app much slower if we deploy with lots of duplicate packages so that's why I'm wondering if we can remove them.